browser-test/src/support/admin_programs.ts

```diff
@@ -453,6 +453,10 @@ export class AdminPrograms {
     ).not.toBeNull()
   }
 
+  async expectStatusSelectorVisible() {
+    await this.page.isVisible('#status-selector')
```

Comment:
Since browser tests are trying to assert things as the user sees them, I think it'd be ideal to avoid doing selections by ID wherever possible.

Instead, could we assert that some text isn't visible on the page?